api/app/src/mappings/withings/body.ts

```diff
@@ -61,6 +63,7 @@ export const mapToBody = (
           };
         }
 
+        defaultWeight["data_source"] = { standardized_type: "MANUALLY ENTERED" };
```

Comment:
re-use const (notice the typo)

Reply:
You're right! Gonna make those right away